### PR TITLE
[MIG] product_packaging_unit_price_calculator: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: product_packaging_unit_price_calculator/wizards/product_package_price.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class ProductPackagePrice(models.TransientModel):
```
Comment (TDu) [inline product_packaging_unit_price_calculator/wizards/product_package_price.py:8]: I see in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

> If there are transient models, you have to add explicit security ACLs for them (and optionally record rules)

Comment (TDu) [reply product_packaging_unit_price_calculator/wizards/product_package_price.py:8]: Also runbot and travis are green, so :shrug:

**Review by TDu — APPROVED** [review]

**Comment by simahawk** [comment]
@dzungtran89  can you please squash fixup commits and drop pot files commits? :pray:

**Comment by dzungtran89** [comment]
hi @simahawk , the PR has been updated

**Pull request merged**